<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtConceptMapRatedConceptDialog</class>
 <widget class="QDialog" name="QtConceptMapRatedConceptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Cluster bij concept: ?</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_complexity">
        <property name="text">
         <string>Complexiteit: ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_concreteness">
        <property name="text">
         <string>Concreetheid: ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_specificity">
        <property name="text">
         <string>Specificiteit: ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_concept_examples">
        <property name="text">
         <string>Voorbeelden/toelichtingen bij concept:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="list_concept_examples">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_cluster_relations">
        <property name="text">
         <string>Relaties bij het cluster:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="list_cluster_relations">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
